src/components/Modal/ModalHeaderWrapper.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render } from '@testing-library/react'
+import * as React from 'react'
+
+import { ModalHeaderWrapper } from './ModalHeaderWrapper'
+
+it('should render children when hasHeader is true', () => {
```

Comment:
I think there are some tests missing:
- hasHeader=true + hasCloseIcon=true
- hasHeader=true + hasCloseIcon=false
- Should call on close (header and close icon)